### ISSUE TITLE
[Easy Dictionary] Unable to translate selected text

### ISSUE THREAD

**Issue by 0x0-00** [raycast/extensions#9350]
### Extension

https://www.raycast.com/isfeng/easydict

### Description

When I select text in a browser or other editor and press the translation shortcut, the selected text is not added to the translation input box, the first time it works, but when I select the text again, the previously translated text is always in the translation box, even if I delete it and use the shortcut to retranslate the selected text, it still doesn't work so I need to quit and wait for a while for it to work

### Steps To Reproduce

1.Select text to translate using shortcut keys
2.Select the new text again and press the shortcut key Translate

**Comment by tisfeng** [comment]
I've debugged it a bit and I feel that Raycast has recently changed the extension activation logic, previously every time I activated an extension using a shortcut key, it would execute the extension code, but now it will decide whether to execute the code or not based on the `Pop to Root Search` option in the settings.

Specifically, if you set 5s, if you activate the extension with the shortcut key within 5s, the extension code will not be executed, only the previous interface will be displayed, if you activate the extension with the shortcut key after 5s, the extension code will be executed, i.e. it will fetch and translate the words.

It's been a while since I've followed Raycast's changes, I don't know why they suddenly changed the logic 😓

<img width="1034" alt="image" src="https://github.com/raycast/extensions/assets/25194972/fa92e37a-72cf-4a0b-b346-907a59d42acd">
